VAMobile/src/screens/HomeScreen/ProfileScreen/ContactInformationScreen/AddressValidation/AddressValidation.tsx

```diff
@@ -22,17 +23,22 @@ export type AddressValidationProps = {
   addressEntered: AddressData
   addressId: number
   snackbarMessages: SnackbarMessages
+  validationData: ValidateAddressData
+  updateAddress: UseMutateFunction<EditResponseData | undefined, unknown, AddressData, unknown>
```

Comment:
Unfortunately, I had to pass the same mutate function from the `EditAddressScreen` so that it can know when to display the loading component when the address is being validated or updated. Even though we could make a new instance of the `useValidateAddress` hook in this component, it doesn't seem like we have a good way to get the loading state of that mutation from the `EditAddressScree`. There is the [useIsMutating](https://tanstack.com/query/latest/docs/react/reference/useIsMutating) hook, but that only returns the number of matching mutation, but no information on the mutations themselves. This could be changing in v5 thought I believe